reference/catalog/brands_catalog.v3.yml

```diff
@@ -3,7 +3,7 @@ info:
   title: Catalog - Brands
   description: |-
   
-    > The Catalog API manages products, categories, brands, bulk pricing rules, and more. To learn more about catalog resources see the [Catalog Overview](/api-docs/store-management/catalog/catalog-overview).
```

Comment:
go ahead and leave this greater than in. then replicate this at the top of each catalog spec file so there's a catalog summary callout at the top of each file.